application/pr/viewer.jsx

```diff
@@ -19,13 +19,17 @@ export default class PRViewer extends React.Component {
         ,   shortName:  null
         ,   num:        null
         ,   groupDetails: []
+        ,   repoId:     null
+        ,   teamcontact: null
         };
     }
     componentDidMount () {
         let owner = this.props.params.owner
         ,   shortName = this.props.params.shortName
         ,   num = this.props.params.num
         ,   groupDetails
+        ,   repoId
+        ,   teamcontact = false
```

Comment:
very minor nit - maybe `isTeamContact` rather than `teamcontact` since this is a boolean?

Reply:
done: https://github.com/w3c/ash-nazg/pull/163/commits/e68920cdeab3a4ebf8aca86cc2e56bc8fd6028a5